docker/docker-compose.dev.yml

```diff
@@ -18,7 +18,7 @@ services:
       SIRIUS_PUBLIC_URL: http://localhost:8080
       FIRM_HUB_HOST: http://localhost:8887
       PREFIX: /supervision/deputies
-      FEATURES: assurance_visits
+      FEATURES:
```

Comment:
Is it worth getting rid of `FEATURES:` as well or do we need it?

Reply:
So I left that in to coincide with the guidance on the README so that future features can be hidden behind this.